client/src/app/(onboarding)/login/login.css

```diff
@@ -4,6 +4,12 @@
     flex-direction: column;
     justify-content: center;
     align-items: center;
+}
+.login-fields{
+   display: flex;
+   flex-direction: column;
+   width: 85%;
```

Comment:
Let's not rely on percentage for setting the width. Add the padding on the container itself if you want the fields to have spacing